service.yml

```diff
@@ -1,9 +1,8 @@
-org_line: International Growth
-director: christianblais
+org_line: Build
+director: tomnewton
```

Comment:
Do you know director directive is necessary? I'm not sure it provides much value having it here for this gem.

Reply:
We can remove. :)

Reply:
👍